dts/common/mcuboot.overlay

```diff
@@ -7,4 +7,12 @@
 	chosen {
 		zephyr,code-partition = &slot0_partition;
 	};
+
+#ifdef CONFIG_BOARD_INTEL_S1000_CRB
```

Comment:
This does not belong here, why isn't this in the board dts file?

Reply:
Removed it here and moved it to the boards dts file. It will be included only when CONFIG_BOOT_MCUBOOTLOADER is defined